packages/content/src/requests/items/proposeAttribute/ProposeAttributeAcceptResponseItem.ts

```diff
@@ -24,7 +24,7 @@ export class ProposeAttributeAcceptResponseItem extends AcceptResponseItem imple
     @validate()
     public attribute: IdentityAttribute | RelationshipAttribute;
 
-    public static override from(value: IProposeAttributeAcceptResponseItem | ProposeAttributeAcceptResponseItemJSON): ProposeAttributeAcceptResponseItem {
+    public static override from(value: IProposeAttributeAcceptResponseItem | Omit<ProposeAttributeAcceptResponseItemJSON, "@type">): ProposeAttributeAcceptResponseItem {
```

Comment:
you should still allow to add `@type`, so `IX | Omit<XJSON, "@type"> | XJSON` would be better here